CONTRIBUTING.md

```diff
@@ -70,3 +70,10 @@ To reach this goal we fixed some rule in our team and we would love to go ahead
 4. After you proposed the PullRequest. If you will receive any mail or find any automatic comment on the Pull Request you opened, it means there is something which is not respecting the project defined code style or your broke any previously created test.
 
 If you respect all these rules you will help us saving time and we will be able to check your pull request faster.
+
+### Naming convention
+
+To keep a clean project, thanks to respect the following naming convention guidelines:
+
+1. Theme attributes use camelCase. If you need to specify variant and size, use the pattern <what><Variant><Size>. ex: buttonPrimaryMedium
+2. Dimens use snakeCase following the pattern vtmn_<what>_<where>_<description>_<size>. ex: vtmn_progressbarcircular_thickness_small
```

Comment:
snakeCase > snake case

Reply:
ex > e.g.:

Reply:
Maybe we should add a glossary for `what`, `where`, `variant`, `size` and `description`?
Note that I don't see the `color` property here but you are using it in this contribution.

Reply:
I think we should add `module` in the name too

Reply:
For example,

`vtmn_topbar_search_contentinsetstart` become `vtmn_appbars_searchbar_contentinsetstart`?

Reply:
Dimens > resources

Reply:
vtmn_<module>_<component>?_<sub_component>?_<description>?_(<size>|<color>)?

Color: `vtmn_navigationbar_icon_color` > `vtmn_appbars_bottomnavigation_icon_color`
Dimen: `vtmn_button_horizontalpadding_medium` > `vtmn_buttons_horizontalpadding_medium`